map/bookmark_manager.cpp

```diff
@@ -64,8 +64,20 @@ std::string const GenerateValidAndUniqueFilePathForKML(std::string const & fileN
 }
 }  // namespace
 
-BookmarkManager::BookmarkManager(GetStringsBundleFn && getStringsBundleFn)
+BookmarkManager::BookmarkManager(GetStringsBundleFn && getStringsBundleFn,
+                                 CreatedBookmarksCallback && createdBookmarksCallback,
```

Comment:
Do we really need three params here? Is it possible to put all callbacks into a single structure, named Callbacks?